pyccel/ast/builtins.py

```diff
@@ -176,6 +176,7 @@ def __new__(cls, arg0, arg1=LiteralFloat(0)):
         return super().__new__(cls, arg0, arg1)
 
     def __init__(self, arg0, arg1 = LiteralFloat(0)):
+        self._shape = arg0.shape if arg0.shape > arg1.shape else arg1.shape
```

Comment:
As we are extending the behaviour of `PythonComplex` beyond what it does in python I don't think we need to handle strange corner cases like this
In addition what happens when `arg0.shape` is `PyccelArraySize(Variable(x),0)`?

Reply:
ah ok.